src/Compilers/VisualBasic/Portable/Syntax/Syntax.xml

```diff
@@ -5084,25 +5084,43 @@ Unify Enum declarations with other block type declarations.
     <!--********************
       -  TupleElementSyntax
       **********************-->
-    <node-structure name="TupleElementSyntax" parent="VisualBasicSyntaxNode">
+    <node-structure name="TupleElementSyntax" abstract="true" parent="VisualBasicSyntaxNode">
       <description>Represents a single declaration of a tuple element. </description>
       <grammar>TupleElement</grammar>
+    </node-structure>
 
-      <node-kind name="TupleElement"></node-kind>
+    <!--********************
+      -  TypedTupleElementSyntax
+      **********************-->
+    <node-structure name="TypedTupleElementSyntax" parent="TupleElementSyntax">
```

Comment:
>TypedTupleElementSyntax [](start = 26, length = 23)

Strictly speaking the other one also can be typed, "UnnamedTupleElementSyntax" might be more precise.